membership_board/README.rst

```diff
@@ -0,0 +1,76 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+================================
+Membership Board for Associations
+================================
+
+Manage membership board and adds tags automaticly to members if they are into
+ it.
```

Comment:
It't not so clear for me. You mean that in a association you could have some members that are part of a board?